### PR TITLE
807 Text brick field standardization - 751 Subtask

### PR DESCRIPTION
**Pull request by robbdavis**
807 Text brick field standardization - 701 Subtask

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_text_brick/templates/fieldable-panels-pane--ucgy_gbl_text_brick_fpp.tpl.php [hunk 1]
```diff
@@ -74,13 +74,15 @@
 <div class="<?php echo $classes; ?>">
     <div class="row <?php print $align; ?>">
         <div class="container">
+          <?php if($content['field_gbl_text_disable_heading']['#items'][0]['value'] == 0): ?>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_text_brick/templates/fieldable-panels-pane--ucgy_gbl_text_brick_fpp.tpl.php]: Add a comment for this check.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_text_brick/ucgy_gbl_text_brick.module [hunk 2]
```diff
@@ -71,6 +71,7 @@ function ucgy_gbl_text_brick_form_alter(&$form, &$form_state, $form_id) {
 
       $form['title']['#title'] = 'Heading (displayed)';
       $form['title']['#description'] = 'This is the heading that will appear at the top of the brick. All headings are required because they are used elsewhere in the system.';
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_text_brick/ucgy_gbl_text_brick.module]: For consistency add a t() around the description field as this is used on some / most of the field standarizations.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_text_brick/templates/fieldable-panels-pane--ucgy_gbl_text_brick_fpp.tpl.php (hunk 1 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_text_brick/templates/fieldable-panels-pane--ucgy_gbl_text_brick_fpp.tpl.php]: Done. I also moved the if statement inside the row div. When it was outside it, contextual links were not accessible when the title was not displayed.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_text_brick/ucgy_gbl_text_brick.module (hunk 2 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_text_brick/ucgy_gbl_text_brick.module]: Done.

**Comment by taeram** [comment]
Closing this PR on GitHub, but keeping the branch around for historical purposes.

**Pull request closed without merging**